### DISCUSSION TITLE
How to Change a language

### DISCUSSION THREAD

**Discussion by ashap5** [1thenikita/Visual-Studio#65] [Q&A] [answered]
How can I change the language from Russian to English? The Language in automaticly set to Russian, although the visual studio is in English.

**Comment by 1thenikita** [comment]
The language is set according to the language of the operating system.

**Reply by ashap5** [reply]
can i somehow change it to english without changing opertaing system language?

**Reply by 1thenikita** [reply, marked as answer]
> can i somehow change it to english without changing opertaing system language?

Yes, but this function is being tested. and will be available in a future implementation.

**Comment by 1thenikita** [comment]
> How can I change the language from Russian to English? The Language in automaticly set to Russian, although the visual studio is in English.

This function is being tested. and will be available in a future implementation
